.travis.yml

```diff
@@ -11,6 +11,7 @@ after_success:
       if [ -n "$TRAVIS_PULL_REQUEST_SHA" ]; then
         npm run travis-deploy-once "npm run deploy-preview -- -d braid-design-system--${TRAVIS_PULL_REQUEST_SHA}.surge.sh";
         npm run travis-deploy-once "npm run post-commit-status";
+        npm run semantic-release -- --dry-run --no-ci;
```

Comment:
You probably shouldn't check this in :trollface: 

(Hey, GitHub finally has a trollface emoji 😍)

Reply:
I really quite liked the dry run of having the version printed out along with the release notes in the build log. Felt clean. But yeah as mentioned, im pulling that out now 😄